tests/acceptance/features/cliCommands/searchReIndex.feature

```diff
@@ -0,0 +1,21 @@
+@env-config
+Feature: reindex space via CLI command
+    As an admin
+    I want to reindex space
+    So that I can improve search performance by ensuring that the index is up-to-date
+
+  Background:
+    Given user "Alice" has been created with default attributes and without skeleton files
+    And the administrator has assigned the role "Space Admin" to user "Alice" using the Graph API
+    And user "Alice" has created a space "new-space" with the default quota using the Graph API
+    And user "Alice" has uploaded a file inside space "new-space" with content "some data" to "textfile.txt"
+
+
+  Scenario: reindex all spaces
+    When user "Alice" reindexes all spaces using the CLI
+    Then the command should be successful
+
+
+  Scenario: reindex a space
+    When user "Alice" reindexes a space with spaceName "new-space" using the CLI
+    Then the command should be successful
```

Comment:
I would check that search works after reindex
maybe we can add step where we found `textfile.txt` file?